dask_geomodeling/geometry/sinks.py

```diff
@@ -202,6 +202,13 @@ def merge_files(path, target, remove_source=False):
                 if source.encoding:
                     kwargs["encoding"] = source.encoding
 
+            # fiona 10 "identifies" json which would result in double "dumping"
```

Comment:
```suggestion
            # fiona 1.10 "identifies" json which would result in double "dumping"
```

There is actually no pin on Fiona in the tests, so how are we sure that it is working with a range of versions? Geoblocks uses 1.9.6 currently